casepro/backend/junebug.py

```diff
@@ -307,5 +336,40 @@ def get_url_patterns(self):
 
         :return: a list of URL patterns.
         """
-        # TODO: Implement views for receiving messages.
-        return []
+        return [
+            url(
+                settings.JUNEBUG_INBOUND_URL, received_junebug_message,
+                name='inbound_junebug_message'
+            ),
+        ]
+
+
+@csrf_exempt
+def received_junebug_message(request):
```

Comment:
At the moment, Junebug doesn't support auth on its requests. You created this issue to solve it though: https://github.com/praekelt/junebug/issues/99 . We could add auth and permissions to this view to future proof, but for now it needs to be turned off..